.github/workflows/backend_checks.yml

```diff
@@ -384,6 +384,12 @@ jobs:
           OP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.OP_SERVICE_ACCOUNT_TOKEN }}
 
           # Secrets to pull from 1Password
+          BIGQUERY_DATASET: op://github-actions/bigquery/BIGQUERY_DATASET
+          BIGQUERY_KEYFILE_CREDS: op://github-actions/bigquery/BIGQUERY_KEYFILE_CREDS
+          DYNAMODB_ACCESS_KEY_ID: op://github-actions/dynamodb/DYNAMODB_ACCESS_KEY_ID
+          DYNAMODB_ACCESS_KEY: op://github-actions/dynamodb/DYNAMODB_ACCESS_KEY
+          DYNAMODB_ASSUME_ROLE_ARN: op://github-actions/dynamodb/DYNAMODB_ASSUME_ROLE_ARN
+          DYNAMODB_REGION: op://github-actions/dynamodb/DYNAMODB_REGION
```

Comment:
if these are only used for the external-datastores tests, can we only pull them if we will use it? idk if the 1pass github action has something like conditional pulling or if we can just add these to the external datastores tests... thinking about the fact that we have a limit on the number of pulls